packages/backend/src/config/parseConfig.ts

```diff
@@ -625,6 +626,7 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                     : parseLoggingLevel(process.env.LIGHTDASH_LOG_FILE_LEVEL),
             filePath: process.env.LIGHTDASH_LOG_FILE_PATH || './logs/all.log',
         },
+        disablePat: process.env.DISABLE_PAT === 'true',
```

Comment:
Suggestion to move this next to the other authentication configuration, inside `auth`.

Reply:
done